### PR TITLE
Homa jumping

### PR DESCRIPTION
**Pull request by Homap**
I suggest jumping in addition to running as well

### CODE REVIEW

File: homa.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@ Hi! LALALALA GOD JUL!
 And Gott Nytt År!
 
 Let's run this year a lot!
-No.
```
Comment (mariasve) [inline homa.txt:5]: Keep No.

File: homa.txt [hunk 2]
```diff
@@ -2,4 +2,6 @@ Hi! LALALALA GOD JUL!
...
+
+
+Let's jump this year!
```
Comment (mariasve) [inline homa.txt:7]: Also No.

**Review by mariasve — COMMENTED** [review]
Some nos.

**Review by Homap — COMMENTED** [review]
No running, no jumping, what a sad year it will be :(

**Review by mariasve — APPROVED** [review]
OK, you jump and run, I cheer you on.

**Pull request merged**